crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -105,19 +115,87 @@ impl TransactionBlockEffects {
     }
 
     /// The reason for a transaction failure, if it did fail.
-    async fn errors(&self) -> Option<String> {
-        match self.native().status() {
-            NativeExecutionStatus::Success => None,
+    /// If the error is a MoveAbort, the error message will be resolved to a human-readable form if
+    /// possible, otherwise it will fall back to simply displaying the Move abort code and location.
```

Comment:
Small wording nits here, because they end up in our GraphQL docs -- `MoveAbort` no longer shows anywhere in our errors because we use the human readable Display form, so it will just say "Abort code", I believe.

```suggestion
    /// If the error is a Move abort, the error message will be resolved to a human-readable form if
    /// possible, otherwise it will fall back to displaying the abort code and location.
```